salon_creator/lib/models/lesson.dart

```diff
@@ -1,45 +1,48 @@
+import 'package:salon_creator/models/resource.dart';
+
 class Lesson {
   String salonId;
   String name;
-  String thumbnail;
-  String media;
+  String thumbnailUrl;
+  String mediaUrl;
   String category;
   String description;
-  bool publish;
-  List<String> resources;
+  bool isPublish;
```

Comment:
```suggestion
  bool isPublished;
```

Sorry for being super picky